docs/deploy/aws.md

```diff
@@ -19,8 +19,11 @@ Expected deployment time: 25 min
 
 {% include_relative includes/prerequisites.md %}
 
-## Creating the Database on AWS RDS
-lakeFS requires a PostgreSQL database to synchronize actions on your repositories.
+## Preparing the Database for the Key Value Store
+lakeFS uses a key-value store to synchronize actions on your repositories. Out of the box, this key-value store can rely on DynamoDB or a PostgreSQL DB. As lakeFS open source, you can also write your own implementation and use any other DB
```

Comment:
Maybe generalize it more and not mention the current supported drivers explicitly, so we won't have to maintain this documentation everytime we add a new driver

Reply:
I guess that if we do add other drivers, we will need to update this doc one way or another, and at least add another configuration section, similar to the 2 we have below.
I think it is best to be as explicit as possible with manuals, so I would vote to leave the specification and expand it in the future (if needed). WDYT?